src/ingest-pipeline/airflow/dags/pas_ftu_segmentation.py

```diff
@@ -315,11 +330,12 @@ def build_cwltool_cmd_ome_tiff_offsets(**kwargs):
         dag_file=__file__,
         retcode_ops=[
             "pipeline_exec_cwl_segmentation",
-            "pipeline_exec_cwl_create_vis_symlink_archive",
             "pipeline_exec_cwl_ome_tiff_offsets",
             "move_data",
         ],
-        cwl_workflows=list(cwl_workflows.values()),
+        cwl_workflows=lambda **kwargs: kwargs["ti"].xcom_pull(
+            key="cwl_workflows", task_ids="build_cmd_ome_tiff_offsets"
```

Comment:
Does this need the workflow_version and workflow_description?
```
        workflow_description=workflow_description,
        workflow_version=workflow_version,
```

Reply:
Good catch!